### PR TITLE
More prebuilt indexes features

### PR DESCRIPTION
**Pull request by qguo96**
- Add `SimpleSearcher.list_prebuilt_indexes()`, `IndexReader.from_prebuilt_index()` and `IndexReader.list_prebuilt_indexes()`
- I use a JSON file to replace the dictionary

### CODE REVIEW

**Comment by qguo96** [comment]
This is a demo of function `list_prebuilt_indexes()`:
Screenshot (1223)
Screenshot (1224)

**Comment by lintool** [comment]
Nice! A few suggestions/comments:

- what if the output is just a pandas dataframe?
- size: just make exact number of bytes? (uncompressed = sum of bytes of all files)?
- `unique terms = -1` - that might be an issue with an underlying method... can you file an issue separate and I'll look into it?
- `pyserini/indexes_info.json`: can we just take the JSON structures and move into directly into Python (i.e., basically make it a `.py' file)? I think this will make PyPI distribution easier, so we don't forget about an additional file...

**Comment by qguo96** [comment]
hi! @lintool, I just created an issue here https://github.com/castorini/pyserini/issues/236.

**Comment by qguo96** [comment]
- Both `uncompressed size` and `compressed size` are displayed in bytes.
- Now the output is a pandas data frame.
- That JSON file is rewritten in a py file.

Here is a demo of output:
Screenshot (1229)
Screenshot (1228)

File: pyserini/index/_base.py [hunk 1]
```diff
@@ -150,6 +151,15 @@ def __init__(self, index_dir):
         self.object = JIndexReader()
         self.reader = self.object.getReader(JString(index_dir))
 
+    @classmethod
```
Comment (lintool) [inline pyserini/index/_base.py:154]: Can we add docstrings to this and below?

**Review by lintool — CHANGES_REQUESTED** [review]
lg. please add docstrings, and we're ready for merging.

File: pyserini/index/_base.py (hunk 1 above)
Comment (qguo96) [reply pyserini/index/_base.py:154]: I add docstrings to `IndexReader` and `SimpleSearcher`. Ready for review.

**Comment by qguo96** [comment]
add information about `enwiki` and `zhwiki`.

**Review by lintool — APPROVED** [review]

**Pull request merged**